storagenode/console/stats.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package console
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Stats is interface for working with node stats db
+type Stats interface {
```

Comment:
The fully qualified name is `console.Stats` so I would expect it to contain statistics about the console.